config/settings.yml

```diff
@@ -1,4 +1,5 @@
 work_dir: tmp/repos
+light_mode: false
```

Comment:
Make it easy for folks to change the colors used depending on their preferred terminal experience.